app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1099,6 +1099,49 @@ class PatientRepositoryAndroidTest {
         .dispose()
   }
 
+  @Test
+  fun verify_deleted_PrescribedDrugs_are_not_included_when_fetching_recent_patients() {
```

Comment:
Rename method to `verify_deleted_prescribed_drugs_are_not_included_when_fetching_recent_patients()`

Reply:
done